instrumentation/jdbi3/src/main/java/brave/jdbi3/RemoteServiceNameResolver.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright The OpenZipkin Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package brave.jdbi3;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class RemoteServiceNameResolver {
```

Comment:
Super minor, I think we shouldn't expose this class publicly:

```suggestion
final class RemoteServiceNameResolver {
```